src/testUtils.ts

```diff
@@ -89,10 +90,36 @@ export function getTestFunctions(doc: vscode.TextDocument, token: vscode.Cancell
 		.then(symbols =>
 			symbols.filter(sym =>
 				sym.kind === vscode.SymbolKind.Function
-				&& (sym.name.startsWith('Test') || sym.name.startsWith('Example')))
+				&& (sym.name.startsWith('Test') || sym.name.startsWith('Example') || isInstanceTestMethod(sym))
+			)
 		);
 }
 
+/**
+ * Checks whether given symbol is a test method of a test suite instance.
+ *
+ * @param symbol Symbol to check.
+ */
+export function isInstanceTestMethod(symbol: vscode.SymbolInformation | string): boolean {
+	let symbolName = typeof symbol === 'string' ? symbol : symbol.name;
+	return testSuiteMethodRe.test(symbolName);
+}
+
+/**
+ * Extracts test method name of a suite test function.
+ * For example a symbol with name "(*testSuite).TestMethod" will return "TestMethod".
+ *
+ * @param symbol Symbol to extract method name from.
+ */
+export function extractInstanceTestName(symbol: vscode.SymbolInformation | string): string {
```

Comment:
All uses of this function passes in a string. So we don't need to support both string and `vscode.SymbolInformation`